.devcontainer/Dockerfile

```diff
@@ -0,0 +1,24 @@
+# [Choice] Ruby version (use -bullseye variants on local arm64/Apple Silicon): 3, 3.1, 3.0, 2, 2.7, 2.6, 3-bullseye, 3.1-bullseye, 3.0-bullseye, 2-bullseye, 2.7-bullseye, 2.6-bullseye, 3-buster, 3.1-buster, 3.0-buster, 2-buster, 2.7-buster, 2.6-buster
+ARG VARIANT=3.1-bullseye
+FROM mcr.microsoft.com/vscode/devcontainers/ruby:0-${VARIANT}
+
+# Install Rails
+RUN gem install rails webdrivers 
+
+# Default value to allow debug server to serve content over GitHub Codespace's port forwarding service
+# The value is a comma-separated list of allowed domains 
+ENV RAILS_DEVELOPMENT_HOSTS=".githubpreview.dev"
+
+# [Choice] Node.js version: lts/*, 16, 14, 12, 10
+ARG NODE_VERSION="lts/*"
```

Comment:
I don't think `node` is required for the app to run.

Reply:
All the devcontainers seem to want to have a version of node installed, my guess is because there are so many CLI tools these days that depend on it, and the devcontainer is oriented around encapsulating the entire development environment, rather than the application environment.  In this case, I'd be inclined to leave this one as a default, in case someone wants to `npm install toolOfChoice`.

Reply:
You don't have to worry about that since I'll reject any PR that adds any JS dependency 😂